src/devices/bus/isa/svga_cirrus.cpp

```diff
@@ -93,15 +202,168 @@ void isa16_svga_cirrus_device::device_reset()
+	ROM_CONTINUE(                                   0x00001, 0x04000 )
+	// Boca Research BRI4610 - ISA16 VGA card, Chips: CL-GD5428-800C-A, RAM: 1MB, Bios Version 1.30B, VESA feature connector, DB15 connector
+	ROM_SYSTEM_BIOS(2, "boca_v1.3", "Boca Research BRI4610 v1.3")
+	ROMX_LOAD("boca_gd5428_1.30b.bin",              0x00000, 0x04000, CRC(6cc8f318) SHA1(2cac4bf2537626cff41aa81ad4627ec6c40dcf99), ROM_SKIP(1) | ROM_BIOS(2) )
+	ROM_CONTINUE(                                   0x00001, 0x04000 )
+	ROM_SYSTEM_BIOS(3, "bri4610_v141a", "Boca Research BRI4610 V1.41a")
+	ROMX_LOAD("isamalaysia.bin" , 0x00000, 0x8000, CRC(b15804d0) SHA1(9778320ec1560e78aa978a4ef21e098535cb0fcf), ROM_BIOS(3) )
+	// Boca Research BRI4611 - ISA16 VGA card - Chip: CL-GD5422-80QC-C - ROM: CL-GD540X/542X VGA BIOS Version 1.00d - RAM: 1MB - Connector: DB15 - VESA feature connector
+	ROM_SYSTEM_BIOS(4, "bri4611", "Boca Research BRI4611")
+	ROMX_LOAD("gd5422_boca.vbi", 0x00000, 0x08000, CRC(dd1de1c2) SHA1(548e84e412563e4dc1c12736a8d355bf91aaa3b9), ROM_BIOS(4) )
+	ROM_SYSTEM_BIOS(5, "ver_1.00d", "Ver. 1.00d")
+	ROMX_LOAD("cl-gd5422_bios_rom_ver_1.00d.bin",   0x00000, 0x04000, CRC(c63b1eb8) SHA1(01782fdef5c22fa6418bc881ee9781bf273c389b), ROM_SKIP(1) | ROM_BIOS(5) )
+	ROM_CONTINUE(                                   0x00001, 0x04000 )
+	// Diamond Speedstar PRO - Chipset: CL-GD5426-80QC-B - ROM: Ver 2.04 - RAM: 1MB - Connector: DB15 - VESA feature connectorVL card
+	ROM_SYSTEM_BIOS(6, "speedstar_pro", "Diamond Speedstar PRO")
+	ROMX_LOAD("diamond5426_v204.vbi", 0x00000, 0x08000, CRC(c98491b0) SHA1(19ae9d2bf39e5b2726d6d556ef2cfa84b06d4da9), ROM_BIOS(6) )
+	// Octek VL-Combo rev.3.2 - This is a Cirrus Logic CL-GD5426 based VGA card that includes 2 serial ports, a parallel port, a floppy controller and an IDE HDD controller.
+	// Chips: CL-GD5426, SMC FDC37C652QF, PROMISE PDC20230C
+	// The 64KB original dump includes two identical 32KB halves with 16KB "quarters" interleaved in each.
+	// The rev.3.3 card uses the same BIOS but a CL-GD5428
+	ROM_SYSTEM_BIOS(7, "octec_vlcombo3.2", "Octek VL-Combo rev.3.2")
+	ROMX_LOAD("octek_vl-combo_rev3_2_27256.bin", 0x00000, 0x04000, CRC(0dbabb0a) SHA1(693c384abf42420276a2727ec4a4c1792704eb30), ROM_SKIP(1) | ROM_BIOS(7) )
+	ROM_CONTINUE(                                   0x00001, 0x04000 )
+	// Octek VL-Combo rev.2.1 - PDC20230-B IDE-controller
+	// Software dump using the MESS method, the Promise HDD-controller part is dumped from D800, the 32K from C800 are FF
+	ROM_SYSTEM_BIOS(8, "octec_vlcombo2.1", "Octek VL-Combo rev.2.1")
+	ROMX_LOAD("octek_vl_combo_v2.1_myc000.bin", 0x00000, 0x08000, CRC(1a97b6d5) SHA1(fde41c45347b66ed4b1f99c3f7057e7411cdf4da), ROM_BIOS(8) )
+	// ROM_LOAD("octek_vl_combo_v2.1_myd800.bin", 0x00000, 0x02000, CRC(2c472bfe) SHA1(ab4879924f5826c38f6066ac769106584ac2d2e8) ) // HDD
```

Comment:
These two belongs to specific ISA card variants. For the time being can you please reorder so that they get the last ROM_BIOS IDs here (i.e. these two to IDs `13` / `14`)?